nfspy/nfspy.py

```diff
@@ -268,12 +268,13 @@ def _gethandle(self, path):
                 fattr = self.rootattr
             else:
                 fh, fattr, cachetime = self.handles[path]
+            # get fattr if it is missing
+            if fattr == None:
```

Comment:
```suggestion
            if fattr is None:
```
See [PEP-8 recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations).